### PR TITLE
open_browser needs to be a Bool(config=True)

### PR DESCRIPTION
**Pull request by robnagler**
Fixes jupyterhub/dockerspawner#88. [docker-stacks/minimal-notebook sets open_browser](https://github.com/jupyter/docker-stacks/blob/3411997451309475812792cd7860e48fe2b4b4e9/minimal-notebook/jupyter_notebook_config.py#L13), and the same `jupyter_notebook_config.py` is used by jupyter/singleuser image.

### CODE REVIEW

File: scripts/jupyterhub-singleuser [hunk 1]
```diff
@@ -171,7 +171,7 @@ class SingleUserNotebookApp(NotebookApp):
     hub_api_url = Unicode(config=True)
     aliases = aliases
     flags = flags
-    open_browser = False
+    open_browser = Bool(False, config=True, help='Here for backwards compatible reasons. Leave as False')
```
Comment (willingc) [inline scripts/jupyterhub-singleuser]: @robnagler Perhaps on the help phrase: `help='Leave parameter set as False. Only change to maintain backwards compatibility.'

**Comment by minrk** [comment]
I don't think this is the right fix. Instead of making it a Bool (it should *not* be configurable, because this is never the right thing to do), the `open_browser = False` config should be removed from the single-user image.

The warning is harmless, and just says that the config is being ignored, as it should be.

**Comment by minrk** [comment]
Alternately, we could make it an enum with a single choice:

```python
open_browser = Enum([False], config=True, help="not really configurable, here to avoid warnings about ignored config")
```

**Comment by robnagler** [comment]
+1 for Enum

**Comment by willingc** [comment]
Agreed @robnagler. I would use Min's suggestion for Enum.

**Pull request closed without merging**